docs/reference/ingest/apis/simulate-ingest.asciidoc

```diff
@@ -0,0 +1,358 @@
+
+[[simulate-ingest-api]]
+=== Simulate ingest API
+++++
+<titleabbrev>Simulate ingest</titleabbrev>
+++++
+
+Executes ingest pipelines against a set of provided documents, optionally
```

Comment:
In this and in the doc for the existing simulate, I'm surprised there is no mention of the simulation/testing aspect. In isolation, this reads as if it actually executes the pipeline (and thus indexes some data).

Reply:
There's a line below that reads `No data is indexed into $Elasticsearch`. I'll add a line about the intended use of this API at the top though.